proposals/aip-1.md

```diff
@@ -0,0 +1,128 @@
+---
+aip: 1
+title: Proposal Purpose and Guidelines
+status: Draft
+type: Process
+author: Scott Willeke (scott@willeke.com)
+created: 2018-04-01
+---
+
+# What is an Agorex Improvement Proposal (AIP)? #
+An AIP is a document providing information to the Agorex community or describing process that the Agorex community follows. The AIP should provide a concise specification of a process and a rationale for it. The AIP author is responsible for building consensus within the community and documenting dissenting opinions.
```

Comment:
This paragraph is a little confusing. I would change it to:

"An AIP is a document providing information to the Agorex community that describes a new process or feature within the organizational structure or the exchange itself. The AIP should provide a concise specification of the feature or process and a rationale for the feature. The AIP author is responsible for building consensus within the community and documenting dissenting opinions."

Reply:
Agreed. Adds more clarity.

Reply:
@JonathonDunford @rowmagnon Thank you for the feedback. I'm fine with this revision. However, I wonder if we want to include "feature" at all though (or at least maybe not now). I found the EIP wording to be fairly unclear for processes. It appears to me the doc was written for features and as an afterthought processes/meta was worked in. I have been thinking of optimizing the wording of AIP-1 to be "process" specific (and toyed with wording of "policy" too). I think feature decisions generally don't require the formality of an AIP and maybe we should focus on processes/policies instead for now. Then later we could add in features if we find that helpful. What are your thoughts?